sdk/servicebus/azure-servicebus/azure/servicebus/common/mixins.py

```diff
@@ -87,9 +87,9 @@ def create_queue(
             dead_lettering_on_message_expiration=dead_lettering_on_message_expiration,
             duplicate_detection_history_time_window=duplicate_detection_history_time_window,
             max_delivery_count=max_delivery_count,
-            enable_batched_operations=enable_batched_operations)
+            enable_batched_operations=enable_batched_operations, fail_on_exist=True)
```

Comment:
This probably merits a docstring as well, at the risk of nitpicking :)  

(Broadly PR looks fine, if I'm really playing the overkill game we could also validate that this flag gets plumbed properly in a unit test.)

Reply:
Thanks @KieranBrantnerMagee .

Docstring updated. Also added same param for other resources(topic, sub)

Reply:
let me see how to make a unittest :)

Reply:
unit test added and passed. could you please review again